src/test/regress/expected/multi_schema_support.out

```diff
@@ -1095,3 +1095,126 @@ SELECT run_command_on_coordinator_and_workers('DROP USER "test-user"');
 (1 row)
 
 DROP FUNCTION run_command_on_coordinator_and_workers(p_sql text);
+-- test run_command_on_* UDFs with schema
+CREATE SCHEMA run_test_schema;
+CREATE TABLE run_test_schema.test_table(id int);
+SELECT create_distributed_table('run_test_schema.test_table','id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- randomly insert data to evaluate below UDFs better
+INSERT INTO run_test_schema.test_table VALUES(1);
+INSERT INTO run_test_schema.test_table VALUES(7);
+INSERT INTO run_test_schema.test_table VALUES(9);
+-- try UDFs which call shard_name as a subroutine
+SELECT run_command_on_placements('run_test_schema.test_table','SELECT pg_table_size(''%s'')');
+    run_command_on_placements     
+----------------------------------
+ (localhost,57637,1190033,t,0)
```

Comment:
This output seems likely to change at some point (e.g. order of items). Could we simplify the output? e.g. `SELECT count(*) FROM run_command_on_placements(...) WHERE success;`